hnn_core/basket.py

```diff
@@ -69,6 +69,7 @@ def __init__(self, gid=-1, pos=-1):
         self._synapse_create()
         self._biophysics()
         self.sect_loc = dict(proximal=['soma'], distal=['soma'])
+        self.record_voltage_soma()
```

Comment:
I'm a little worried that this isn't the option of turning this off (the default in HNN GUI). Would it be possible to use a new parameter `record_vsoma` in lieu of the HNN  GUI parameter `save_vsoma`?

Reply:
This is all that's needed to add a parameter:
https://github.com/jonescompneurolab/hnn-core/pull/184/files#diff-7fab1444eb40a87689cc4afa7c7aa15c80dc4a18a2179d3e231185de704c3bcfR138

Reply:
I think a parameter to the `__init__` of the cell class should work

Reply:
Given all the issues that are cropping up with the increased data flow through stderr, I agree this will be important to disable by default (despite the negligible impact on simulation time). 

I'll combine both suggestions. `record_vsoma` will be set to `False` in the default params. `network_builder.py` will pass this value to the `__init__` of `cell.py` and a logical statement will check and return an empty list.

Reply:
I've implemented it such that `params['vsoma']` is pulled out when the cells are built in `network_builder.py`. Setting `record_vsoma` = 0 makes `cell.rec_v` creates an empty `h.Vector()` in place of a recording.